### PR TITLE
Add detail page for artifacts and job.

### PR DESCRIPTION
**Pull request by splattael**
## Warning: Untested code ahead

I've just added detail pages for builds and jobs.
### Repo

Each build displayed on the repo page is now linked to the specific build (e.g. #13 is linked).

Repo
### Build details

The new build page display all jobs for this build. Each job links to the specific job detail page.

Builds
### Job details

Display job details including job build log.

Job
### Finally

The code in this PR is not tested and needs your feedback.

If you like the idea of this PR I'm happy to add specs and such.

### CODE REVIEW

**Comment by splattael** [comment]
This PR is related #3

**Comment by sarahhodne** [comment]
Awesome, I'll merge this in locally and add some tests to it before I push it.

**Comment by sarahhodne** [comment]
Just read the last line of your pull request. If you want to add specs, please do. I'll look over the code once more and give you some feedback.

File: lib/travis/lite/templates/job.mustache [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+{{#job}}
+<div>
+  <ul>
+    <li>Repo: <a href="/{{slug}}">{{slug}}</a></li>
+    <li>Build <a href="/{{slug}}/builds/{{build_id}}">#{{build_number}}</a></li>
+    <li>Status: <span class="label label-{{status_class}}">{{status}}</li>
+    <li>Branch: {{branch}}</li>
+    <li>Language: {{config_label}}</li>
+    <li>Compare <a href="{{compare_url}}">on github</a></li>
```
Comment (sarahhodne) [inline lib/travis/lite/templates/job.mustache]: This should be "GitHub" and not "github". See @kneath's [tweet](https://twitter.com/kneath/status/275841609151700992).

**Comment by sarahhodne** [comment]
Code looks good. There are some naming issues, but they're mostly in the code I created earlier, so I'll just update everything later to remain consistent.

**Comment by splattael** [comment]
@henrikhodne I've added some specs.

I've moved `format_build_status` and friends to `Layout`. Repos are now being marking yellow when running.

You talked about "naming issues". Which one? :) I guess I misnamed `Matrix` which is might be called `Artifact` instead.

File: lib/travis/lite/templates/job.mustache (hunk 1 above)
Comment (kneath) [inline lib/travis/lite/templates/job.mustache]: :heart_eyes_cat:

**Comment by sarahhodne** [comment]
@splattael The naming issues I was actually the `API` methods not having the "type" of resource being returned in the name, but I must've misread the code since I don't see it anymore.

`Matrix` should be `Matrix` in my opinion, `Artifact` is a completely different thing (currently the only artifacts are logs).

Thanks for the PR!

**Comment by sarahhodne** [comment]
Just deployed this. Thanks again!

**Pull request merged**